adapter/config/ingress/controller.go

```diff
@@ -62,10 +62,10 @@ func NewController(client kubernetes.Interface, mesh *proxyconfig.ProxyMeshConfi
 	informer := cache.NewSharedIndexInformer(
 		&cache.ListWatch{
 			ListFunc: func(opts meta_v1.ListOptions) (runtime.Object, error) {
-				return client.ExtensionsV1beta1().Ingresses(options.Namespace).List(opts)
+				return client.ExtensionsV1beta1().Ingresses(meta_v1.NamespaceAll).List(opts)
```

Comment:
As discussed, there could be issues if multiple ingress controllers are deployed, like when multiple people run tests in the same time, and they all claim all ingress resources. This will not be the case in production, but it may affect our testing.

I will pass a flag from the test to limit the ingress to a single namespace.